plugins/todo-backend/src/service/TodoReaderService.ts

```diff
@@ -75,7 +72,7 @@ export class TodoReaderService implements TodoService {
       );
     }
 
-    const url = this.getEntitySourceUrl(entity);
+    const url = getEntitySourceLocation(entity).target;
```

Comment:
This removes the check that `type === 'url'`, we don't want to be trying to fetch contents for other location types. Let's add the check back here, along with the check that the location exists